docs/usage/configuration/config_documentation.md

```diff
@@ -1768,6 +1768,30 @@ Example configuration:
 max_image_pixels: 35M
 ```
 ---
+### `prevent_media_downloads_from`
```

Comment:
I was just thinking about the name. This is a block list, if you later want to use the concept with allow lists, you have to rename the parameter.

Wouldn't it be better to adapt the name to the current parameters for allowing and disallowing federation or url previews?

Reply:
Allow lists aren't currently planned for this feature - I'd favour a config migration (deprecation) later on if we decide to go that route.